src/gc/gcinterface.h

```diff
@@ -868,6 +868,11 @@ class IGCHeap {
     // Unregisters a frozen segment.
     virtual void UnregisterFrozenSegment(segment_handle seg) = 0;
 
+#if defined(FEATURE_BASICFREEZE) && defined(VERIFY_HEAP)
```

Comment:
This is changing GC/EE interface. We should bump the version at the top of the file to avoid LocalGC mismatches.

Reply:
@Maoni0 bumped it earlier today.